src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<Scheme>
+
+@section BreadCrumbs {
+    <a href="/" class="govuk-back-link">Home</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h1 id="comparison-results-h1" class="govuk-heading-l">
+            Comparison of schemes
+        </h1>
+        <table id="comparison-results-table" class="govuk-table">
+            <thead class="govuk-table__head">
+                <tr>
+                    <th scope="col" class="govuk-table__header">
+                        Scheme
+                    </th>
+                    <th scope="col" class="govuk-table__header">
+                        Cost
+                    </th>
+                    <th scope="col" class="govuk-table__header">
+                        Benefits
+                    </th>
+                    <th scope="col" class="govuk-table__header">
+                        Time
+                    </th>
+                </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+                @foreach(Scheme scheme in Model) {
+                    <tr>
+                        <td class="govuk-table__cell">
+                            <a id="scheme-header-link-@scheme.HtmlId" asp-route="schemes" asp-route-schemeUrl="@scheme.Url" class="govuk-link">@scheme.Name</a>
+                        </td>
+                        <td class="govuk-table__cell">
+                            @scheme.ShortCost
+                        </td>
+                        <td class="govuk-table__cell">
+                            @scheme.ShortBenefits
+                        </td>
+                        <td class="govuk-table__cell">
+                            @scheme.ShortTime
+                        </td>
+                    </tr>
+                }
+            </tbody>
+        </table>
+    </div>
```

Comment:
We need a 'Return to list of all schemes' link under the table.

Reply:
Done in latest commit